test/integration/controller_test.go

```diff
@@ -275,6 +276,7 @@ func newTestController(t *testing.T) (
 		serviceCatalogSharedInformers.Instances(),
 		serviceCatalogSharedInformers.Bindings(),
 		brokerClFunc,
+		&record.FakeRecorder{},
```

Comment:
is there something we can check for? Otherwise we're not really testing anything